x/treasury/handler.go

```diff
@@ -66,12 +80,19 @@ func handleMsgOracleMint(ctx sdk.Context, k keeper.Keeper, msg types.MsgOracleMi
 		return err.Result()
 	}
 
-	ctx.EventManager().EmitEvent(
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeMsgSend,
```

Comment:
This should be a new event type `EventTypeOracleMint`

Reply:
ok. Modified the code accordingly